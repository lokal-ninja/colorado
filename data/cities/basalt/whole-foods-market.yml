id: 7834153200
type: node
latitude: 39.38702
longitude: -107.08271
postcode: "81621"
city: Basalt
street: "Reed Street"
housenumber: 340
phone: "1-970-927-1500"
opening_hours: "Mo-Su 08:00-21:00"
website: "https://www.wholefoodsmarket.com/"
