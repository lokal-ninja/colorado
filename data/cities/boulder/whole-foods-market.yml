id: 564835927
type: node
latitude: 40.02383
longitude: -105.25603
postcode: "80301"
city: Boulder
street: Pearl Street
housenumber: "2905"
phone: +1 303-545-6611
opening_hours: Mo-Su 07:00-22:00
website: https://www.wholefoodsmarket.com/stores/boulder
