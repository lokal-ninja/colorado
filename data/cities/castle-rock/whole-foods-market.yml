id: 1015956813
type: node
latitude: 39.42383
longitude: -104.87745
postcode: "80108"
city: Castle Rock
street: Promenade Parkway
housenumber: "6384"
phone: +1 720-673-4700
opening_hours: Mo-Su 08:00-21:00
website: https://www.wholefoodsmarket.com/stores/castlerock
