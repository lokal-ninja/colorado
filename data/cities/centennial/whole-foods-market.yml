id: 1340617555
type: node
latitude: 39.59211
longitude: -104.96182
postcode: "80122"
city: Centennial
street: "South York"
housenumber: 6853
phone: "+1 303-996-3660"
opening_hours: "Mo-Su 08:00-21:00"
website: "https://www.wholefoodsmarket.com/stores/southglenn"