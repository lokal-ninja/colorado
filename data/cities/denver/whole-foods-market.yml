id: 2424886792
type: node
latitude: 39.6520158
longitude: -104.9017556
postcode: "80231"
city: Denver
street: "East Hampden Avenue"
housenumber: 7400
phone: "303-488-2000"
opening_hours: ""
website: "https://wholefoodsmarket.com/"
