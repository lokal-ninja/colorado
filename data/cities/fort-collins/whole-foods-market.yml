id: 333519611
type: way
latitude: 40.5586
longitude: -105.07839
postcode: "80525"
city: Fort Collins
street: "South College Avenue"
housenumber: 2201
phone: ""
opening_hours: "Mo-Su 07:00-22:00"
website: ""
