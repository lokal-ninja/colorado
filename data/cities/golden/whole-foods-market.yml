id: 2967816324
type: node
latitude: 39.73658
longitude: -105.16148
postcode: "80401"
city: Golden
street: West Colfax Avenue
housenumber: "14357"
phone: +1-303-277-1339
opening_hours: Mo-Su 08:00-21:00
website: https://www.wholefoodsmarket.com/stores/colfax
